### DISCUSSION TITLE
[FR] sharing support for fail2ban, when DMS runs in HAPROXY mode with nginx-ingress in kubernetes

### DISCUSSION THREAD

**Discussion by IvanI3** [docker-mailserver/docker-mailserver#3073] [Show and tell]
# Feature Request
Not actually a FR, just sharing my simple script & config if this could be useful for someone...

## Context
This is a fail2ban action and some supporting configs, which allow to use fail2ban when docker-mailserver runs in Kubernetes container behing nginx-ingress TCP proxy, using HAPROXY mode.
This setup doesn't allow to use built-in fail2ban with iptables action, because DMS container does't see real IP addresses at TCP level (DMS works in HAPROXY mode). So, one of the way to block annoying IPs - use nginx-ingress resource.

### What have you done already?
Action script, configs and instructions are provided [here](https://github.com/IvanI3/nginx-ingress-configmap).

**Comment by georglauterbach** [comment]
I converted your issue into a discussion as this better fits the topic :)

**Reply by audioscavenger** [reply]
can be closed, the link is dead anyways
